Sources/Player/Asset.swift

```diff
@@ -26,6 +26,12 @@ final class ResourceLoadedPlayerItem: AVPlayerItem {
 }
 
 /// An asset representing content to be played.
+///
+/// Three categories of assets are provided:
+///
+/// - Simple ``simple(url:metadata:configuration:)`` assets which can be played directly.
+/// - Custom ``custom(url:delegate:metadata:configuration:)`` assets which require a custom resource loader delegate.
+/// - Encrypted ``encrypted(url:delegate:metadata:configuration:)`` assets which require a FairPlay content key session delegate.
```

Comment:
Proposal since there are several methods for each category:

1. Only mention the categories without prototypes:

```suggestion
///
/// Three categories of assets are provided:
///
/// - Simple assets which can be played directly.
/// - Custom assets which require a custom resource loader delegate.
/// - Encrypted assets which require a FairPlay content key session delegate.
```

2. Use a DocC extension to organize methods in these three categories.